generated/proto/policy.proto

```diff
@@ -33,4 +33,23 @@ message Retention {
 message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
+	repeated AggregationType aggregation_types = 3;
```

Comment:
Would it make sense to separate this from the policy message to have a 1-to-1 mapping with the structs in the policy package?

Was thinking about the case where we might want to define policies in the future without aggregation types (e.g., say we perform aggregation on the query side and the aggregated metric doesn't go through the aggregation tier so the agg types are not needed)